src/plugins/intel_cpu/tests/functional/single_layer_tests/detection_output.cpp

```diff
@@ -111,6 +115,21 @@ class DetectionOutputLayerCPUTest : public testing::WithParamInterface<Detection
         return result.str();
     }
 
+    // There are two major implemenation path for small tensor and large tensor(confidence tensor) compared with cache size.
+    // This function is to scale input shape based on current machine cache size,
+    // such that tensor size is large enough to cover sparsity implementation.
+    static void scale_input_shapes(std::vector<ov::test::InputShape> &inShapes) {
```

Comment:
I would strongly advise to avoid such cache size dependency in tests. Can we just add more test cases with different shapes to make sure sparse impl will be covered on each system?

Reply:
done, thanks Dmitry.